### PR TITLE
feat: use time buffer by default instead timestamp key

### LINKED ISSUE #39: do not chunk on timestamp
Currently by default this plugin will chunk the logs based on timestamp. https://github.com/newrelic/newrelic-fluentd-output/blob/be9232133b18023f000d7097411934832fe5020c/lib/fluent/plugin/out_newrelic.rb#L40 This means it will spam the logging API based upon the granularity of your timestamps. For example, if your timestamps have the granularity of seconds, it will send logs every second. This is apparently too much for the server to handle at scale, so it basically drops everything.

Instead, this plugin should default to chunking over a larger time window. Or at least this should be mentioned in the README. https://docs.fluentd.org/configuration/buffer-section#time

### PR DESCRIPTION
**Pull request by danybmx**
This PR set the default buffer to time instead to the timestamp field.
It also adds a "playground" docker environment so it could be easy to test use cases.

Fixes #39

### CODE REVIEW

**Comment by noly** [comment]
Solves [#LOGGING-5205](https://newrelic.atlassian.net/browse/LOGGING-5205)

File: lib/newrelic-fluentd-output/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module NewrelicFluentdOutput
-  VERSION = "1.2.0"
+  VERSION = "1.3.0"
```
Comment (noly) [inline lib/newrelic-fluentd-output/version.rb:2]: Hi 👋 wondering if these change should be a Patch. What do You think 🤔

File: playground/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## Writing logs
+
+Since the file `./testlogs/test.log` are mounted as a volume on the docker instance and
+is being tailed by fluentd you could write some text to the file and it should be picked
+up by fluentd.
+
+This command should write a line on the log file and it will reach new relic in a while.
+
+`echo "Hello world" >> ./testlogs/test.log`
```
Comment (noly) [inline playground/README.md:20]: Could You add the `awk` command as well?

File: lib/newrelic-fluentd-output/version.rb (hunk 1 above)
Comment (danybmx) [reply lib/newrelic-fluentd-output/version.rb:2]: Yup, I agree, a patch/bugfix bump should be enough, thanks!

**Comment by danybmx** [comment]
Changes done @noly! could you take a look? I've also added the issue bug template.

Thanks for the suggestions! 🙇🏻

**Review by noly — APPROVED** [review]
yass

**Comment by noly** [comment]
Great work Dani!! 🙇‍♀️

**Pull request merged**